build-livecd-root

```diff
@@ -21,8 +21,9 @@ fi
 
 trap cleanup EXIT
 
+KEEP_TMPDIR=yes
```

Comment:
I'd rather keep this to `no` as a default value. I build ten images a day and `/tmp` is by default a memory mapped folder...